lib/open_api_spex/operation2.ex

```diff
@@ -0,0 +1,93 @@
+defmodule OpenApiSpex.Operation2 do
```

Comment:
Does what `Operation` does, but uses `Cast.cast/3` instead of `Schema.cast/3` and `Schema.validate/3`.